Presentation.Web/app/components/org/user/org-user-delete.controller.ts

```diff
@@ -242,11 +244,18 @@
             if (!confirm('Er du sikker på, at du vil slette brugeren?')) {
                 return;
             }
-            this.userRoleAdministrationService.removeUser(this.loggedInUser.currentOrganizationId, this.userToModify.Id)
-                .then(success => {
-                    if (success) {
-                        this.$uibModalInstance.close();
+
+            this.userService.getUserDeletionStrategy(this.userToModify.Id)
```

Comment:
as you can see you are missing the org context.. we are administering org rights here